choppa/IO/input.py

```diff
@@ -218,21 +227,35 @@ def extract_ligands(system):
         """[Placeholder] Returns a system's ligands"""
         return system
 
-    def check_validity(self, complex):
+    def reset_complex_sequence(self, complex):
+        """Adjusts protein sequence indexing to run from 1 to n, rather than whatever wonky indexing
+        the crystallographer may have come up with. BioPython can do this but there are some
+        protections built in against hard re-indexing multi-chain proteins. By first setting
+        the indexing super high (starting at 100,000) and then re-indexing starting from 1 we can
+        circumvent these protections. For more details see https://github.com/biopython/biopython/pull/4623
```

Comment:
We can install from  `git+` source if you want this functionality in the aforementioned biopython PR. Only issue is we would then be blocked on release upstream until biopython merges and does a release.

Reply:
Perhaps its just better to use this hack.